src/message/__tests__/messageActions-test.js

```diff
@@ -16,93 +16,23 @@ const streamNarrowStr = JSON.stringify(streamNarrowObj);
 
 describe('messageActions', () => {
   describe('doNarrow', () => {
-    test('when no messages in new narrow and caughtUp is false, actions to fetch messages and switch narrow are dispatched', () => {
+    test('when async `doNarrow(...)` is dispatched, actions to switch narrow are dispatched', () => {
       const store = mockStore<GlobalState, Action>(
         eg.reduxState({
+          accounts: [eg.selfAccount],
           session: { ...eg.baseReduxState.session, isHydrated: true },
-          caughtUp: {
-            [streamNarrowStr]: {
-              newer: false,
-              older: true,
-            },
-          },
           ...navStateWithNarrow(HOME_NARROW),
-          narrows: {},
-          streams: [eg.makeStream({ name: 'some stream' })],
-        }),
-      );
-
-      store.dispatch(doNarrow(streamNarrowObj));
-      const actions = store.getActions();
-
-      expect(actions).toHaveLength(3);
-      const [action0, action1, action2] = actions;
-      expect(action0.type).toBe('DO_NARROW');
-      expect(action1.type).toBe('MESSAGE_FETCH_START');
-      if (action1.type === 'MESSAGE_FETCH_START') {
-        expect(action1.narrow).toEqual(streamNarrowObj);
-      }
-      expect(action2.type).toBe('Navigation/PUSH');
-    });
-
-    test('when no messages in new narrow but caught up, only switch to narrow, do not fetch', () => {
```

Comment:
Probably not great to remove these tests without something to replace them—it seems like these are mostly testing work that's the job of `isFetchNeededAtAnchor`? So, maybe I should write some tests for `isFetchNeededAtAnchor`.

Reply:
OK, done.